BinEncoder.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+
+from MessagersInterfaces import Listener, Notifier, T
+from RossServer import *
+from UMDDecoder import *
+
+
+class RossEventToBin(Listener[RossEvent], Notifier[bytes]):
+    def __init__(self, listener: Listener[bytes], num_of_cam=127):
+        '''
+        Creates "bytes" with lenght of 33
+        1-st byte - lenght of "bytes"
+        Others - camera's states
+        '''
+        self._listener = listener
+        self._a = bytes()
+        self._a = [0] * 33
+        self._a[0] = 32
+
+    def on_message(self, message: RossEvent, notifier: Notifier[RossEvent]):
+        '''
+        Fills the bytes with the corresponding state
+        of the corresponding camera
+        Param: RossState
+        Returns: bytes
+        '''
+        multi = 0
+        place = 0
+        for i in range (1, self._a[1] + 1, 1):
```

Comment:
PEP8 - Whitespace in Expressions and Statements: Avoid extraneous whitespace in the following situations: Immediately before the open parenthesis that starts the argument list of a function call